nodejs/week2/src/backend/routes/meal.js

```diff
@@ -0,0 +1,17 @@
+const { request, response } = require("express");
+const express = require("express");
+const { number } = require("yargs");
+const router = express.Router();
+const meals = require("../data/meals.json");
+
+// params.. meal with specific id
+router.get("/meal/:id", (request,response)=> {
+    const mealId = request.params.id;
+    meals.forEach((meal) => {
+		if (meal.id === parseInt(mealId)) {
```

Comment:
What if there are no meal with the id that the user wants to get? 
You can implement an `else` condition here for that.